src/plugins/guided_onboarding/server/routes/config_routes.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { IRouter } from '@kbn/core/server';
+import { API_BASE_PATH } from '../../common/constants';
+import { guidesConfig } from '../helpers/guides_config';
+
+export const registerGetConfigRoute = (router: IRouter) => {
+  // Fetch the config of the guide
+  router.get(
+    {
+      path: `${API_BASE_PATH}/configs`,
```

Comment:
Do we need to send the config for all guides? Or can we just return the one that's needed based on the guide id?

Reply:
First, I thought that it would make sense to just get all existing guides at once because they don't change and if `guideId` changes we don't need to send another request. But now I'm thinking that usually within one browser session, the user probably will only interact with one guide. And we still have to fetch the config on each page reload. So I'll update the logic here.